src/main/scala/org/mdedetrich/stripe/v1/Plans.scala

```diff
@@ -53,18 +53,19 @@ object Plans extends LazyLogging {
   /**
     * @see https://stripe.com/docs/api#plan_object
     * @param id
-    * @param amount              The amount in cents to be charged on the interval specified
+    * @param amount              The amount in cents to be charged on the interval specified.
     * @param created
     * @param currency            Currency in which subscription will be charged
     * @param interval            One of [[Interval.Day]], [[Interval.Week]], [[Interval.Month]] or
-    *                            [[Interval.Year]]. The frequency with which a subscription
-    *                            should be billed.
+    *                            [[Interval.Year]]. The frequency with which a subscription should
+    *                            be billed.
     * @param intervalCount       The number of intervals (specified in the [[interval]]
     *                            property) between each subscription billing. For example,
     *                            \[[interval]]=[[Interval.Month]] and [[intervalCount]]=3
     *                            bills every 3 months.
     * @param livemode
-    * @param name                Display name of the plan
```

Comment:
There was no `name` parameter for plan, but it's child object had a `name` parameter for the service product.